Cargo.toml

```diff
@@ -137,7 +137,7 @@ rand_distr = "0.4.0"
 regex = "1.3.9"
 bytes = { version = "0.5.6", features = ["serde"] }
 stream-cancel = "0.6.2"
-hyper = "0.13"
+hyper = "0.13.10"
```

Comment:
Unfortunately `0.13.10` not include `a00cc20afc597cb55cbc62c70b0b25b46c82a0a6`..
`0.13.0` commits: https://github.com/hyperium/hyper/commits/v0.13.10
`0.13.x` commits: https://github.com/hyperium/hyper/commits/0.13.x
`hyper` will be updated in #6217

Reply:
well that sucks! thanks for the heads up!

Reply:
Thanks, that's a good find! Thanks, great job!

Reply:
I moved `hyper` update from #6217 to separate PR for faster merge: https://github.com/timberio/vector/pull/6407 Will be grateful for approval there.